fx_opt: 4.681405334472656250e+02
fy_opt: 4.677453308105468750e+02
cx_opt: 4.291349145581207267e+02
cy_opt: 2.545160360476984351e+02
